test/acceptance/_helpers.bash

```diff
@@ -117,7 +117,7 @@ wait_for_ready() {
     for i in $(seq 60); do
         if [ -n "$(check ${POD_NAME})" ]; then
             echo "${POD_NAME} is ready."
-            sleep 2
+            sleep 10
```

Comment:
This could make the tests really slow as our test cases grow in number and complexity. Wondering if there's a better way to manage this other than `sleep`?

Reply:
I can be smarter here with `kubectl wait`.

Reply:
A possible implementation could be something like,

```shell
  VAULT_READY=1
  while [ $VAULT_READY != 0 ]; do
    kubectl -n <some namespace> wait --for condition=available statefulset/vault > /dev/null 2>&1
    VAULT_READY="$?"
  done
```

Reply:
That's what I was thinking @billimek! 👍

Reply:
Very good